aten/src/ATen/native/Convolution.cpp

```diff
@@ -1896,7 +1896,7 @@ std::tuple<Tensor,Tensor,Tensor> _convolution_double_backward( const std::option
         auto i_size = input.sizes();
         for (const auto i : c10::irange(2, gI_size.size())) {
```

Comment:
Do we not have a utility for an int64 c10::range? Seems like an oversight if so.